src/SFML/Window/iOS/SFAppDelegate.mm

```diff
@@ -151,6 +162,36 @@ - (void)applicationWillTerminate:(UIApplication *)application
     }
 }
 
+- (bool)supportsOrientation:(UIDeviceOrientation)orientation
+{
+    if (!self.sfWindow)
```

Comment:
Empty lines after return statements? Never seen someone doing that, but that's what would stop this PR, its not a problem to add them.